WordPress/Sniffs/VIP/ValidatedSanitizedInputSniff.php

```diff
@@ -82,6 +82,9 @@ public function register() {
 	 */
 	public function process_token( $stackPtr ) {
 
+		$notValidated = false;
+		$notSanitized = false;
```

Comment:
Just wondering if the code would be easier to read if these variables would be reversed in meaning, i.e. 
```php
$validated = true;
$sanitized = true;
```

Also wondering whether the default should be `true` (or `false` as the current code stands), though changing that might mean more extensive changes to the logic of the sniff are needed.